types/extrinsic_signature.go

```diff
@@ -25,7 +25,7 @@ type ExtrinsicSignatureV3 struct {
 }
 
 type ExtrinsicSignatureV4 struct {
-	Signer    Address
```

Comment:
this will break for all the chains without the multiaddress.

Reply:
Good point, there are a couple of chains also using substrate master who for reasons of simplicity prefer not to use MultiAddress.

We could this solve this by making `ExtrinsicSignatureV4.Signer` an interface type:

```
type ExtrinsicSignatureV4 {
    Signer interface{}
    ...
}
```

Reply:
I think we could abstract out the Signerwith Address and MultiAddress as default implementations. This wouldn't break the compatibility